blog/secadvisor.md

```diff
@@ -0,0 +1,363 @@
+Go the "Generator" page and specify the source and destination nodes between which to generate the network traffic, as well as the characterization of the flow you want to generate.
+
+### Observe Security Advisor objects being created
+
+Objects are created in the Object Store only if there is flow information to be saved.
+
+The output log of secadvisor should show that an object is being sent to the object store about once per minute.
+Check your bucket in your object store to verify that you see a new object about once per minute.
+Stop the secadvisor or stop captures in the GUI to stop the creation of objects in the object store.
+
+### Content of Security Advisor objects
+
+The Security Advisor saves flow information to an object store in GZIP compressed JSON encoding format.
+Each object created by the Security Advisor contains the flows that were captured and filtered according to the parameters set in the secadvisor.yml configuration file.
+An example of entries in an unzipped object created by Security Advisor might be the following:
+
+```
+{
+	"UUID": "7521e85422cbc042",
+	"LayersPath": "Ethernet/IPv4/TCP",
+	"Version": "1.0.8",
+	"Status": "UPDATED",
+	"Network": {
+		"Protocol": "IPV4",
+		"A": "169.45.67.210",
+		"B": "169.44.184.135",
+		"B_Name": "0_0_kube-sjc03-crc3a0aa10c90641aeb0e7740ee137c40b-w1_0"
+	},
+	"Transport": {
+		"Protocol": "TCP",
+		"A": "32396",
+		"B": "4723"
+	},
+	"LastUpdateMetric": {
+		"ABPackets": 0,
+		"ABBytes": 112,
+		"BAPackets": 0,
+		"BABytes": 178,
+		"Start": 1551199625889,
+		"Last": 1551199655889
+	},
+	"Metric": {
+		"ABPackets": 2,
+		"ABBytes": 178,
+		"BAPackets": 3,
+		"BABytes": 244,
+		"Start": 0,
+		"Last": 0
+	},
+	"Start": 1551199609804,
+	"Last": 1551199631044,
+	"updateCount": 1,
+	"NodeType": "device"
+}
+```
+
+For each flow we have the following fields and types, all collected in JSON encoding format.
+
+```
+        UUID             string
+        LayersPath       string
+        Version          string
+        Status           string
+        FinishType       string
+        Network          *SecurityAdvisorFlowLayer
+        Transport        *SecurityAdvisorFlowLayer
+        LastUpdateMetric *flow.FlowMetric
+        Metric           *flow.FlowMetric
+        Start            int64
+        Last             int64
+        UpdateCount      int64
+        NodeType         string
+```
+
+UUID - unique ID of the flow.
+
+LayersPath - represents the layers of the network stack; typical values might be "Ethernet/IPv4/TCP", "Ethernet/IPv4/UDP/DNS","Ethernet/IPv4/ICMPv4".
+
+Version - version of the Security Advisor code being run.
+
+Status - status of the flow; may be "STARTED", "ENDED", or "UPDATED".
+
+FinishType - indicates how a flow ended; may be "SYN_FIN", "SYN_RST", "Timeout", "OVERFLOW", or empty.
+
+Network - provides network layer information including Protocol (e.g. IPV4), source and destination addresses.
```

Comment:
breakdown to fields:
- Network - network layer infromation
  - Protocol - protocol according to standard port assignment
  - A - source address
  - B - destination address

Reply:
added detail of sub-fields